src/engraving/dom/tremolosinglechord.cpp

```diff
@@ -68,7 +68,7 @@ TremoloSingleChord::~TremoloSingleChord()
     //
     // delete all references from chords
     //
-    if (chord()) {
+    if (chord() && chord()->tremoloSingleChord() == this) {
```

Comment:
For those who wonder (like me, initially) why not instead of this change `Chord::setTremoloSingleChord` does something like this:
```c++
    if (m_tremoloSingleChord) { // *before* m_tremoloSingleChord = tr;
        m_tremoloSingleChord->setChord(nullptr);
    }
```
In the first place, `setChord` does not exist, and instead one has to use `setExplicitParent`; in the second place, that will break undoability.

In the case of TremoloTwoChord, that may be a bit different, but why not follow the same logic for consistency.